### ISSUE TITLE
[Bug]: Date selector in schedules for desktop app doesn't handle dates in a 6th row correctly

### ISSUE THREAD

**Issue by mullermn** [actualbudget/actual#3701]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue
- [X] I will be providing steps how to reproduce the bug (in most cases this will also mean uploading a demo budget file)

### What happened?

This might be a bit niche, but it's reproducible on my system.
On the MacOS desktop app (ie, non in a browser) it is not possible to select any date that causes the date selector to overflow in to a 6th row of days.

Examples as below are September and December 2024 where the month starts at the end of the week causing there to be 6 rows of days rather than 5. When any day in the 6th row is clicked the date selector closes with no date selected as if the click had been completely outside the date selector.
![image](https://github.com/user-attachments/assets/2aca3838-3cbe-429a-87dd-1126a75dff88)
![image](https://github.com/user-attachments/assets/8bc1ae43-241c-4dee-af5b-587b27d6f11b)

For contrast, a 5 row month works as expected.
![image](https://github.com/user-attachments/assets/14aef353-7b38-46f5-865b-ae99b0d85fea)

This does not happen in Chrome or Safari on MacOS. Can't test on mobile as schedules are not available there. Date selectors in other areas of the application (eg. transaction entry) seem to work fine 

### Where are you hosting Actual?

Pikapods

### What browsers are you seeing the problem on?

Desktop App (Electron)

### Operating System

Mac OSX

**Comment by youngcw** [comment]
This is working for me in the appimage.  Can you check if its still broken on Mac?

**Comment by mullermn** [comment]
Yes, this is still the case - checked December 2024 and March 2026. I noticed in testing that the selected date (eg 31/03/2026) does briefly appear in the 'From' box, but it is apparently lost after that point so I guess it's the first stage (white) date popup that's misbehaving rather than the blue calendar popup.

This is client/server 25.6.1

**Comment by dnarimot** [comment]
Hi! I'm a student in the CodePath AI301 program and I'd like to work on this issue. I'll start by reproducing the problem locally this week. Any pointers on where to start in the codebase would be appreciated!

**Comment by PuddleOfFat** [comment]
I am not able to reproduce this issue. I am on Mac Desktop app (no server). MacOS 26.5.2. Apple M3 Pro

**Comment by mullermn** [comment]
I have just retested this on v26.7.0, electron client and pikapods server (same setup as for the original report) and it works fine now, so this can be closed from my perspective.
